### ISSUE TITLE
v1.16.1 is boken

### ISSUE THREAD

**Issue by markussiebert** [db-operator/charts#36]
sorry - its late but want you to know ...

no values set - fails with:

```
Helm install failed for release infra-database/db-operator with chart db-operat │
│ or@1.16.1: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors: │
│   line 47: mapping key "description" already defined at line 45                                            │
│   line 48: mapping key "type" already defined at line 46 
```

version 1.15.1 is working

**Comment by allanger** [comment]
Hi, will be fixed by https://github.com/db-operator/charts/pull/32

**Comment by allanger** [comment]
Should be fixed in 1.16.2: https://github.com/db-operator/charts/releases/tag/db-operator-1.16.2
